draft-ietf-scitt-architecture.md

```diff
@@ -331,12 +331,15 @@ Before an Issuer is able to produce Signed Statements, it must first create its
 A DID can be *resolved* into a *key manifest* (a list of public keys indexed by a *key identifier*) using many different DID methods.
 
 Issuers MAY choose the DID method they prefer, but with no guarantee that all Transparency Services will be able to register their Signed Statements.
-To facilitate interoperability, all Transparency Service implementations SHOULD support the `did:web` method {{DID-WEB}}.
+To facilitate interoperability, all Transparency Service implementations MUST support the `did:web` method {{DID-WEB}}.
```

Comment:
Create a new issue to move this to a new place!

Reply:
Ping the issue link here to resolve this, thread